lib/player.rb

```diff
@@ -0,0 +1,26 @@
+class Player
+
+  attr_reader :name, :user_input
+
+  def initialize(name, display)
+    @name = name
+    @display = display
+  end
+
+  def rock
+    "rock"
+  end
+
+  def paper
+    "paper"
+  end
+
+  def scissors
+    "scissors"
+  end
+
+  def take_player_choice
```

Comment:
When naming your methods, think about the class as the object that is doing the action. 
In this case, the class is player and the method should reflect an action that the player is doing.
This method here is named as though something else is getting the choice from the player. I think if the player is the one _doing_ then a method name like `choose`, or something like that, might be more expressive.
Then you can called the method in a way that sounds nice, like `player_one.choose()`